### ISSUE TITLE
Multischedulers don't work

### ISSUE THREAD

**Issue by yetanotherape** [ui/rq-scheduler#167]
It seems that now you can't run two schedulers on two servers for high availability. 

This happens because before the call of `acquire_lock()` it still runs `register_birth()` which raises `ValueError("There's already an active RQ scheduler")`. Initially `register_birth()` and `register_death()` were deleted in #143, but then they were reverted back for some reason.

**Comment by oxalorg** [comment]
True. Also the current locking mechanism doesn't seem to be as robust and may give rise to a possibility of multiple schedulers acquiring the lock.

**Comment by jsam** [comment]
We are experiencing same issue in our rolling updates, so I'm just wondering if there is progress/work being made on this topic?

**Comment by oxalorg** [comment]
@jsam This has been fixed by #212 (which is now merged to master). Could you perhaps give it a try and let us know if it solved your use case?
